tests/tests-query-filters.php

```diff
@@ -37,7 +37,7 @@ public function test_edd_block_attachments_no_attachment_bail() {
 	public function test_edd_block_attachments_no_parent_bail() {
 
 		// Prepare test
-		$filename 		= '../assets/images/loading.gif';
+		$filename 		= EDD_PLUGIN_DIR . '/assets/images/loading.gif';
```

Comment:
Looking into my failing unit tests more, I'm not seeing this change in `master` or in `release/3.0`, only in this merge branch. Changing it back to the relative path resolves the erroring tests, but I don't know why these two are erroring and the other four are not.

Reply:
I changed it because without this change, unit tests were actually failing for me. :thinking: I'd also seen it failing in Buildkite and thought this might fix it. Example of a failure: https://buildkite.com/sandhills-development-llc/easy-digital-downloads/builds/339#6269c2a1-3913-4385-91a6-3a2be152d4bb

I don't know why those suddenly started failing when they were fine before. :shrug:

Reply:
I'll remove this change and we can move it to a new issue https://github.com/easydigitaldownloads/easy-digital-downloads/issues/8473